mlflow/server/auth/sqlalchemy_store.py

```diff
@@ -0,0 +1,102 @@
+from typing import List, Optional
+from sqlalchemy import (
+    Column,
+    String,
+    ForeignKey,
+    Integer,
+    Boolean,
+)
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.orm import relationship, sessionmaker
+from werkzeug.security import generate_password_hash, check_password_hash
+
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import RESOURCE_ALREADY_EXISTS
+from mlflow.store.db.base_sql_model import Base
```

Comment:
I think this results in the auth database containing unnecessary tables such as experiments. Can we create a separate `Base` class and use it?

Reply:
We can verify this by using the following code:

```python
import sqlite3

sqlite_file = "basic_auth.db"
conn = sqlite3.connect(sqlite_file)
c = conn.cursor()
c.execute("SELECT name FROM sqlite_master WHERE type='table';")
print(c.fetchall())
# -> [('experiments',), ('registered_models',), ('users',), ('runs',), ('experiment_tags',), ('model_versions',), ('registered_model_tags',), ('registered_model_aliases',), ('experiment_permissions',), ('registered_models_permissions',), ('tags',), ('metrics',), ('latest_metrics',), ('params',), ('model_version_tags',)]
```

Reply:
you're right, that was my bad